universal-application-tool-0.0.1/test/controllers/HomeControllerAuthenticatedTest.java

```diff
@@ -30,8 +36,24 @@ protected Application provideApplication() {
 
   @Before
   public void setUp() {
+    // Get the config, and hack it so that all requests appear authorized.
     Config config = app.injector().instanceOf(Config.class);
-    config.setClients(new Clients(AnonymousClient.INSTANCE));
+    AnonymousClient client = AnonymousClient.INSTANCE;
```

Comment:
question: what's the relationship between a client an account? does an account always have one and only one client?

Reply:
Think "client" like "IDCSClient" - it's an interface to whatever your authentication / authorization system is.  So it's more like clients retrieve or create "profiles", which is pac4j's name for what you're calling an account.  Multiple clients could interact with the same profile - that's probably what we'll do for secure links or other no-creds login stuff.